recipes/guile/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "v2.1.2" %}
+
+package:
+  name: guile
+  version: {{ version }}
+
+source:
+  git_url: http://git.savannah.gnu.org/r/guile.git
+  git_tag: f9490cdb3fd9c99de6976077cb87ef77c591967a
+
+build:
+  number: 0
+  skip: True # [win]
```

Comment:
Two space between selector and previous text please (as below).

      skip: True  # [win]